src/librustc_lint/builtin.rs

```diff
@@ -89,12 +89,6 @@ impl LintPass for WhileTrue {
 }
 
 declare_lint! {
-    UNSIGNED_NEGATION,
-    Warn,
-    "using an unary minus operator on unsigned type"
-}
```

Comment:
It would be nice to print a "deprecated lint" warning for existing code using `#![allow(unsigned_negation)]`, instead of an unknown_lint warning.